boards/shields/adafruit_2_8_tft_touch_v2/Kconfig.defconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_ADAFRUIT_2_8_TFT_TOUCH_V2
+
+if DISPLAY
+
+config ILI9340
```

Comment:
ILI9340 can only be enabled if SPI is enabled, so possibly the default value for SPI needs to be set to `y`